developer/config.yaml

```diff
@@ -6,10 +6,10 @@
 cluster_type: openshift
 
 # git_url refers to a git repo to be considered as the source of truth for Argo CD applications.
-git_url: https://github.com/openshift-pipelines/pipeline-service.git
+git_url: https://github.com/AndrienkoAleksandr/pipeline-service.git
```

Comment:
Just an note that we'll have to be careful not to merge any references to your repo/branch when merging the PR.